floater/generators.py

```diff
@@ -27,7 +27,7 @@ def xyz_to_geo(xyz_coord):
 class FloatSet(object):
     """Represents a set of initial float positions on a regular grid."""
 
-    def __init__(self, xlim, ylim, dx=1., dy=1.):
+    def __init__(self, xlim, ylim, dx=1., dy=1., model_grid=None):
```

Comment:
We now pass `model_grid` as an optional keyword argument when initializing the FloatSet object.